.azure-pipelines/run-test-elastictest-template.yml

```diff
@@ -159,7 +159,11 @@ steps:
 
           # Else, internal build image repo, download from internal sonic-mgmt repo
           else
-            curl -u $(AZP_REPO_ACCESS_TOKEN) "${{ parameters.MGMT_URL }}&commitOrBranch=${{ parameters.MGMT_BRANCH }}&api-version=5.0-preview.1&path=.azure-pipelines%2Fpr_test_scripts.yaml" -o ./.azure-pipelines/pr_test_scripts.yaml
+            if [ -z "$(MSSONIC-TOKEN)" ]; then
```

Comment:
how can we get MSSONIC-TOKEN without adding sonicbld group?

Reply:
We will add this group where the PR template is used. Here, do not add it because it's in a public repo.